GitUI/CommandsDialogs/SettingsDialog/Pages/GitConfigSettingsPage.cs

```diff
@@ -133,118 +137,87 @@ protected override void PageToSettings()
             }
         }
 
-        private void GlobalMergeTool_TextChanged(object sender, EventArgs e)
+        private string BrowseDiffMergeTool(string toolName, string path)
         {
-            if (IsLoadingSettings)
+            var diffMergeToolConfig = _diffMergeToolConfigurationManager.LoadDiffMergeToolConfig(toolName, null);
```

Comment:
If the path does not exist (like if the tool is not configured in GE), exception occurs

Reply:
Could you please provide me with steps to reproduce?

![image](https://user-images.githubusercontent.com/4403806/69527155-ccf50000-0fbf-11ea-9c8e-6f23f705fe1e.png)

Reply:
Found the issue, thank you!

With a fix will open the `Program Files` folder, if any issues:
![image](https://user-images.githubusercontent.com/4403806/69527856-3f1a1480-0fc1-11ea-907f-866aa603e017.png)